From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Piotr Szczepanik <piter75@gmail.com>
Date: Mon, 5 Oct 2020 23:17:55 +0200
Subject: [ARCHEOLOGY] Switched RockPro64 to mainline u-boot (#2218)

> X-Git-Archeology: > recovered message: > * Switched RockPro64 to mainline u-boot
> X-Git-Archeology: > recovered message: > * Enabled rockusb and ums for RockPro64 in mainline u-boot
> X-Git-Archeology: > recovered message: > * Switched RockPro64 u-boot to TPL/SPL/BLOB scenario
> X-Git-Archeology: > recovered message: > * Enabled advanced recovery button support for RockPro64
> X-Git-Archeology: > recovered message: > * Disabled SDR104 mode for RockPro64 in legacy
> X-Git-Archeology: - Revision f2db96252530d2f5585755b518e7fd020b5f0392: https://github.com/armbian/build/commit/f2db96252530d2f5585755b518e7fd020b5f0392
> X-Git-Archeology:   Date: Mon, 05 Oct 2020 23:17:55 +0200
> X-Git-Archeology:   From: Piotr Szczepanik <piter75@gmail.com>
> X-Git-Archeology:   Subject: Switched RockPro64 to mainline u-boot (#2218)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
index f10e1bf0e2a6..83bc06cf1306 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dts
@@ -898,7 +898,7 @@
 	cap-sd-highspeed;
 	disable-wp;
 	num-slots = <1>;
-	sd-uhs-sdr104;
+	// sd-uhs-sdr104;
 	vqmmc-supply = <&vcc_sd>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&sdmmc_clk &sdmmc_cmd &sdmmc_cd &sdmmc_bus4>;
-- 
Armbian

